hr_grade_rank/views/hr_grade_rank_view.xml

```diff
@@ -8,59 +8,55 @@
             <form string="Employees Grade">
                 <sheet>
                     <group>
-                        <field name="name" required="1" />
+                        <field name="name" required="1"/>
                     </group>
                     <notebook>
                         <page string="Ranks">
                             <field name="rank_ids">
-                                <tree>
-                                    <field name="name" />
-                                    <field name="salary_range" />
-                                    <field name="description" />
-                                </tree>
+                                <list>
+                                    <field name="name"/>
+                                    <field name="salary_range"/>
+                                    <field name="description"/>
+                                </list>
                             </field>
                         </page>
                     </notebook>
-                    <field name="description" placeholder="Description" />
+                    <field name="description" placeholder="Description"/>
                 </sheet>
             </form>
         </field>
     </record>
 
     <!-- GRADE TREE VIEW -->
-    <record id="view_employee_grade_tree" model="ir.ui.view">
-        <field name="name">grade.grade.tree</field>
+    <record id="view_employee_grade_list" model="ir.ui.view">
```

Comment:
No need to replace tree with list in the ID of any view. as it may cause issues if define reference of the view based on ID.